DomainContent/Rust/DJ-Tools/DJ-Tools_Spawner.js

```diff
@@ -0,0 +1,560 @@
+            entID = makeBox(
+                name,                 
+                boxPos, 
+                vec(BOX_WIDTH, BOX_HEIGHT, BOX_DEPTH), 
+                color, 
+                stringified
+            );
+            allEnts.push(entID);
+        });
+    }
+
+    function createDJInputPadModels() {
+        [LEFT, RIGHT].forEach(function (side) {
+            var name,
+                entID,
+                modelPos,
+                rotation,
+                url,
+                stringified,
+                userData = {},                
+                DISTANCE_LEFT = 0.52,
+                HEIGHT = 0,
+                DISTANCE_BACK = -0.70,
+                MODEL_WIDTH = 0.4,
+                MODEL_HEIGHT = 0.05,
+                MODEL_DEPTH = 0.4;
+
+            if (side === LEFT) {
+                modelPos = Vec3.sum(
+                    phlashProps[1].position, 
+                    vec(DISTANCE_LEFT, HEIGHT, DISTANCE_BACK)
+                );
+                url = particlePadLeftModel;
+            } else {
+                modelPos = Vec3.sum(
+                    phlashProps[1].position, 
+                    vec(-DISTANCE_LEFT, HEIGHT, DISTANCE_BACK)
+                );
+                url = particlePadRightModel;
+            }
+            
+            name = "Set_Phlash_Pad_Models_" + side;
+            rotation = Quat.fromPitchYawRollDegrees(0, 180, 0);
+            userData.grabbableKey = { grabbable: false };
+            stringified = JSON.stringify(userData);
+            entID = makeModel(
+                name,                 
+                modelPos,
+                vec(MODEL_WIDTH, MODEL_HEIGHT, MODEL_DEPTH), 
+                rotation,                
+                url,
+                stringified
+            );
+            allEnts.push(entID);
+        });
+    }
+
+    function createDJInputZone() {
+        var name,
+            entID,
+            zonePos,
+            stringified,       
+            userData = {},
+            HEIGHT = 0.0,
+            DISTANCE_BACK = -0.9,
+            ZONE_WIDTH = 2,
+            ZONE_HEIGHT = 2,
+            ZONE_DEPTH = 1.3;
+
+        zonePos = Vec3.sum(
+            phlashProps[1].position, 
+            vec(0, HEIGHT, DISTANCE_BACK)
+        );
+
+        name = "Set_Phlash_Pad_Zone";
+        userData.grabbableKey = { grabbable: false };        
+        stringified = JSON.stringify(userData);
+        entID = makeZone(
+            name,             
+            zonePos, 
+            vec(ZONE_WIDTH, ZONE_HEIGHT, ZONE_DEPTH), 
+            stringified
+        );
+        allEnts.push(entID);
+    }
+
+    // Main
+    deleteIfExists();
+    if (DEBUG) {
+        createDebugCube();
+    }
+    createDJInputZone();
+    createDJInputPadSensors();
+    createDJInputPadModels();
+    createDJEndpointParticles();
+    createDJEndpointLights();
+
+    // Cleanup
+    function scriptEnding() {
+        console.log("### in script ending");
```

Comment:
Remove console logging or wrap this into a debug statement

Reply:
Done